src/extractors/custom/medium.com/index.js

```diff
@@ -42,28 +40,34 @@ export const MediumExtractor = {
 
         const $img = $node.find('img').slice(-1)[0];
         const $caption = $node.find('figcaption');
+
         $node.empty().append([$img, $caption]);
       },
+
+      // Remove any smaller images that did not get caught by the generic image
+      // cleaner (author photo 48px, leading sentence images 79px, etc.).
+      img: $node => {
+        const width = parseInt($node.attr('width'), 10);
+        if (width < 100) $node.remove();
+      },
     },
 
     // Is there anything that is in the result that shouldn't be?
     // The clean selectors will remove anything that matches from
     // the result
-    clean: [],
+    clean: ['span', 'svg'],
   },
 
   date_published: {
-    selectors: [['time[datetime]', 'datetime']],
+    selectors: [['meta[name="article:published_time"]', 'value']],
   },
 
   lead_image_url: {
     selectors: [['meta[name="og:image"]', 'value']],
   },
 
   dek: {
-    selectors: [
-      // enter selectors
-    ],
+    selectors: [['meta[name="og:description"]', 'value']],
```

Comment:
Another minor note on dek: Does `og:description` match what's on the page as the dek? This was a long conversation, but basically we decided we shouldn't fetch a dek if it doesn't explicitly exist on the page, visually.